build.sbt

```diff
@@ -1,8 +1,9 @@
 // This build is for this Giter8 template.
 // To test the template run `g8` or `g8Test` from the sbt session.
 // See http://www.foundweekends.org/giter8/testing.html#Using+the+Giter8Plugin for more details.
-lazy val root = (project in file(".")).
-  settings(
+lazy val root = (project in file("."))
+  .enablePlugins(SbtPlugin)
```

Comment:
This was the reason that the scripted tests didn't work - I hadn't enabled this plugin apparently. :-)